src/mps/abstractmps.jl

```diff
@@ -1465,6 +1465,12 @@ function (::Type{MPST})(
   return M
 end
 
+function (::Type{MPST})(
+  A::Array{ElT}, sites; kwargs...
+) where {MPST<:AbstractMPS,ElT<:Number,N}
```

Comment:
No need for `ElT` or `N` here. Also we could make this accept any `AbstractArray` (the `ITensor` constructor may not work with general `AbstractArray` but that is a work in progress).